Decompiler/Options.cs

```diff
@@ -43,6 +43,10 @@ class Options
             HelpText = "Verify checksums and signatures.")]
         public bool VerifyVPKChecksums { get; set; }
 
+        [Option('c', "vpk_cache", DefaultValue = false,
```

Comment:
Can you remove `c` and only make the full name available? This option is not useful enough to have a shorthand.